sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -796,3 +797,71 @@ case class SchemaOfJson(
 
   override def prettyName: String = "schema_of_json"
 }
+
+/**
+ * A function which returns all the keys of outer JSON object.
```

Comment:
Since JSON can have a nested structure with multiple `inner`s and `outer`s, shall we use `the outmost` instead of `outer`?